pkg/spec/spec.go

```diff
@@ -31,14 +31,17 @@ type Function struct {
 
 // FunctionSpec contains func specification
 type FunctionSpec struct {
-	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
-	Runtime  string             `json:"runtime"`
-	Type     string             `json:"type"`
-	Topic    string             `json:"topic"`
-	Schedule string             `json:"schedule"`
-	Deps     string             `json:"deps"`
-	Template v1.PodTemplateSpec `json:"template" protobuf:"bytes,3,opt,name=template"`
+	Handler     string             `json:"handler"`      // Function handler: "file.function"
+	Function    string             `json:"function"`     // Function file content or URL of the function
+	ContentType string             `json:"content-type"` // Function file content type
```

Comment:
This does not take the same values/structure as HTTP Content-Type header, so should be renamed (or changed to _be_ the same as HTTP Content-Type).